content/en/docs/tutorials/acme/migrating-from-kube-lego.md

```diff
@@ -88,16 +88,21 @@ the value of the `LEGO_SECRET_NAME` environment variable.
 You should download a copy of this secret resource and save it in your local
 directory:
 
+Install [yq](https://kislyuk.github.io/yq/#installation) then run the command below:
```

Comment:
I found two versions of `yq` when I googled for it. This seems to be the original version and according to my colleague @maelvls, it is the better of the two : https://github.com/jetstack/cert-manager/pull/3996#issue-643103680

The other tool that might be even better is `kubectl neat` which is mentioned in this SO article: https://stackoverflow.com/questions/61392206/kubectl-export-is-deprecated-any-alternative 
and in https://github.com/kubernetes/kubernetes/issues/90066

In that issue they also complain about the `managedFields` content which is apparently ommitted by `kubectl get` since v1.21 https://github.com/kubernetes/kubernetes/pull/96878

Reply:
Unfortunately with my current setup I'm unable to verify the behavior of the tools with the `managedField.
The cluster is still on `1.18.20` and my `kubectl` version is `1.19.13`.

`kubectl neat` is a krew plugin and krew needs to be installed separately.
It feels an overkill for me compared to a simple yq installation.

What would be your preference?